### ISSUE TITLE
Change your converter Image Sizes

### ISSUE THREAD

**Issue by somerandomuser1** [EmmanuelBeziat/bns-profile#14]
you have the wrong image sizes (or atleast for NA), the correct size is 351x576

**Comment by EmmanuelBeziat** [comment]
Hi,

If this is only for NA (as it seems to be), I'll have to add a select option on the website.

Thanks for letting me know that.

**Comment by somerandomuser1** [comment]
my apologies. for some reason when i try to take a new character photo, it is 351x576, and then when i use THAT image as the base and try to change it, it always fails to upload. but if i use an older working image that is already 378x620 from a long time ago it will work just fine. idk if they changed something or not, but either way, the ingame character shots are actually 351x576

**Comment by EmmanuelBeziat** [comment]
Yeah, something definitely happened: all the pictures in my BnS folder have been somehow changed from `.jpg` to `.thm`. I don’t know what they did, but it’s really weird.

Anyway, I’ll update the app very soon, but at the moment I won’t have B&S under my hand until monday, so I prefer to wait untill then, so I can test before updating.

Thanks!
